File: API/Web.AppCore/Entities/News.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Web.AppCore.Entities
{
    /// <summary>
    /// Tin tức 
    /// Created BY : NVHIEN 9/8/2021
    /// </summary>
    public class News
    {
        /// <summary>
        /// Id khóa chính
        /// </summary>
        public Guid? NewsId { get; set; }
        /// <summary>
        /// Tiêu đề tin tức
        /// </summary>
        public string NewsTitle { get; set; }
        /// <summary>
        /// Nội dung tin tức
        /// </summary>
        public string NewsContent { get; set; }
        /// <summary>
        /// Tin hot
        /// </summary>
        public int NewsHot { get; set; }
        /// <summary>
        /// Tin tức về sản phẩm
        /// </summary>
        public int NewsProduct { get; set; }
        /// <summary>
        /// Tin tực sale
        /// </summary>
        public int NewsSale { get; set; }
        /// <summary>
        /// Tin tức về sự kiện
        /// </summary>
        public int NewsEvent { get; set; }
        /// <summary>
        /// Link image
        /// </summary>
        public int NewsHome { get; set; }
        /// <summary>
        /// Link image
        /// </summary>
        public string LinkImage { get; set; }

        /// <summary>
        ///  Thời gian tạo
        /// </summary>
        public DateTime? CreatedDate { get; set; }
        /// <summary>
        /// Thời gian sửa đổi
        /// </summary>
        public DateTime? ModifiedDate { get; set; }
    }
}

File: testRest/src/main/java/com/example/testRest/Models/News.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package com.example.testRest.Models;

import java.time.LocalDateTime;
import java.util.Date;
import javax.persistence.*;
import java.util.UUID;
import org.hibernate.annotations.Type;
/**
 *
 * @author BOSS
 */
@Entity
public class News {
    @Id
    @GeneratedValue(generator="system-uuid")
    @Column(length = 36)
    @Type(type = "uuid-char")
    private UUID newsId;
    
    @Column
    private String newsTitle;
    
    @Column
    private String newsContent;

    @Column
    private String linkImage;

    @Column
    private int newsHot;
    
    @Column
    private int newsProduct;
    
    @Column
    private int newsSale;
    
    @Column
    private int newsEvent;
    
    @Column
    private int newsHome;
    
    @Column
    private LocalDateTime createdDate;
    @Column
    private LocalDateTime modifiedDate;

    public UUID getNewsId() {
        return newsId;
    }

    public void setNewsId(UUID newsId) {
        this.newsId = newsId;
    }

    public String getNewsTitle() {
        return newsTitle;
    }

    public void setNewsTitle(String newsTitle) {
        this.newsTitle = newsTitle;
    }

    public String getNewsContent() {
        return newsContent;
    }

    public void setNewsContent(String newsContent) {
        this.newsContent = newsContent;
    }

    public String getLinkImage() {
        return linkImage;
    }

    public void setLinkImage(String linkImage) {
        this.linkImage = linkImage;
    }
    /////
    public int getNewsHot() {
        return newsHot;
    }

    public void setNewsHot(int newsHot) {
        this.newsHot = newsHot;
    }
    /////
    public int getNewsProduct() {
        return newsProduct;
    }

    public void setNewsProduct(int newsProduct) {
        this.newsProduct = newsProduct;
    }
    /////
    public int getNewsSale() {
        return newsSale;
    }

    public void setNewsSale(int newsSale) {
        this.newsSale = newsSale;
    }
    /////
    public int getNewsEvent() {
        return newsEvent;
    }

    public void setNewsEvent(int newsEvent) {
        this.newsEvent = newsEvent;
    }
    /////
    public int getNewsHome() {
        return newsHome;
    }

    public void setNewsHome(int newsHome) {
        this.newsHome = newsHome;
    }
    /////
    public LocalDateTime getCreatedDate() {
        return createdDate;
    }

    public void setCreatedDate(LocalDateTime createdDate) {
        this.createdDate = createdDate;
    }

    public LocalDateTime getModifiedDate() {
        return modifiedDate;
    }

    public void setModifiedDate(LocalDateTime modifiedDate) {
        this.modifiedDate = modifiedDate;
    }
}
